eggs1.rst

```diff
@@ -184,7 +176,7 @@ Before we can use our new package we have to tell Plone about it. Edit ``buildou
     test-eggs +=
         ploneconf.site [test]
 
-This tells Buildout to add the egg ``ploneconf.site``. Since it is also in the `sources` section buildout will not try to download it from pypi but will expect it in ``src/ploneconf.site``. *fs* allows you to add packages on the filesystem without a version control system, or with an unsupported one.
+This tells Buildout to add the egg ``ploneconf.site``. Since it is also in the `sources` section buildout will not try to download it from pypi but will expect it in ``src/ploneconf.site``. *fs* allows you to add eggs on the filesystem without a version control system, or with an unsupported one.
```

Comment:
I am not sure, why we need to rename packages with eggs here.
The same goes for distribution/package in the code above.
What is the general idea here, or did I miss some best practices from the official plone docs?